components/RegisterForm.js

```diff
@@ -187,15 +362,18 @@ const DivChildRadioButton = styled.div`
 const DetailsFieldset = styled.fieldset`
   display: flex;
   width: 100%;
-  align-items: center;
-  justify-content: flex-start;
+  flex-direction: column;
   gap: 1rem;
   border: none;
 `;
 
 const InputFirstName = styled.input`
   width: 100%;
   border: none;
+
+  :focused {
```

Comment:
The pseudo class should be named `:focus` and not `:focused`.
```suggestion
  :focus {
```